proto/api/v1/metadata_service.proto

```diff
@@ -71,25 +71,25 @@ message DatabaseBackupStatus {
 // The initial intended use is to disable certain functionality that does not make sense in the Cloud Service context.
 message CentralServicesCapabilities {
     enum CapabilityStatus {
-        // "Unknown" means "not disabled".
-        // The reason it's not called "Enabled" is that extended checks might be required to confirm true ability to use
-        // e.g. container AWS IAM role or GCP workload identity, and we can't positively say "Enabled" while such checks
-        // aren't implemented.
-        // This means the user should be allowed to use the capability (both via UI and API) but an attempt may not be
-        // successful if the corresponding configuration does not match the actual environment.
-        Unknown = 0;
+        // "Maybe" means "not disabled".
+        // The reason it's not called "Available" is that extended checks might be required to confirm true ability to
+        // use e.g. container AWS IAM role or GCP workload identity, and we can't positively say "Available" while such
+        // checks aren't implemented.
+        // "Maybe" means the user should be allowed to use the capability (both via UI and API) but an attempt may not
+        // be successful if the corresponding configuration does not match the actual environment.
+        Maybe = 0;
 
         // Capability is disabled, meaning the corresponding UI should be disabled and attempts to use related APIs
         // should lead to errors.
-        Disabled = 1;
+        CapabilityDisabled = 1;
```

Comment:
I don't have a strong opinion on what terminology we should use here, but here's my weak opinion:

1. Status names should be uniform:  either both start with `Capability...` or both don't. I would argue that keeping `Capability` as a prefix would make statuses more searchable.
2. I think keeping the words `Disabled` and `Enabled` in names is important - that's terminology that is more known to most developers. If you want to express that it is only marked as enabled, perhaps something among the lines `MarkedAsEnabled`/`MarkedAsDisabled` or `TreatAsEnabled`/`TreatAsDisabled`?

Saying all that, I think you already spent much more time than me thinking about the correct wording here, so if you have a strong opinion about why it should stay like this feel free to keep the names as they are.

Reply:
Changed to `CapabilityAvailable` / `CapabilityDisabled`. Hope it would make sense for the future generations.